packages/fx-core/src/common/samples-config.json

```diff
@@ -42,6 +42,16 @@
             "configuration": "Ready for local debug",
             "suggested": true
         },
+        {
+            "id": "graph-toolkit-one-productivity-hub",
+            "title": "One Productivity Hub using Graph Toolkit",
+            "shortDescription": "One Productivity Hub app of Microsoft Teams Tab app.",
```

Comment:
This seems not exactly the same with what Ayca provided, is this updated version?

Reply:
I follow the decryption of "graph toolkit contact exporter" sample.

Reply:
1. Description short: Microsoft Teams tab with SSO to view calendar events, to-do tasks and file folders.
2. Description long: One Productivity Hub sample shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit components and TeamsFx provider. You can run this app locally or deploy it to Microsoft Azure.****

Reply:
thanks, updated